### PR TITLE
Update version: martinchrzan.ColorPicker version 1.2.1

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115130&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/m/martinchrzan/ColorPicker/1.2.1/martinchrzan.ColorPicker.installer.yaml [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
+Platform:
+- Windows.Desktop
+InstallerType: msi
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: --silent
```
Comment (Trenly) [inline manifests/m/martinchrzan/ColorPicker/1.2.1/martinchrzan.ColorPicker.installer.yaml:15]: image

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/m/martinchrzan/ColorPicker/1.2.1/martinchrzan.ColorPicker.installer.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/m/martinchrzan/ColorPicker/1.2.1/martinchrzan.ColorPicker.installer.yaml:15]:
```suggestion
  Silent: /quiet
  SilentWithProgress: /passive
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

It appears that the provided installer switches aren't functional.

File: manifests/m/martinchrzan/ColorPicker/1.2.1/martinchrzan.ColorPicker.installer.yaml [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
+ProductCode: '{83946ED6-6198-4987-A4CF-F7F49AD3DA8E}'
+ReleaseDate: 2023-02-26
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/martinchrzan/ColorPicker/releases/download/1.2.1/Setup.msi
+  InstallerSha256: 46E302CF877A57424E954C248E91D738DD88C1DB361EE3499FF79B523C7FEDFA
+  InstallerSwitches:
+    Silent: --silent
+    SilentWithProgress: --silent
```
Comment (stephengillie) [inline manifests/m/martinchrzan/ColorPicker/1.2.1/martinchrzan.ColorPicker.installer.yaml:25]:
```suggestion
    Silent: /quiet
    SilentWithProgress: /passive
```

File: manifests/m/martinchrzan/ColorPicker/1.2.1/martinchrzan.ColorPicker.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/m/martinchrzan/ColorPicker/1.2.1/martinchrzan.ColorPicker.installer.yaml]: I'm not sure why this change didn't "take".

**Review by stephengillie — APPROVED** [review]

**Pull request merged**